agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/common/MqttOptionService.java

```diff
@@ -4,6 +4,7 @@
 import com.dke.data.agrirouter.api.env.Environment;
 import com.dke.data.agrirouter.impl.EnvironmentalService;
 import com.dke.data.agrirouter.impl.common.ssl.KeyStoreCreationService;
+import de.dev4agriculture.dto.RouterDevice;
```

Comment:
Internal package declaration?

Reply:
Uups, extraction error 🙈

Fixed, thanks for mentioning